examples/iota_lens/analysis_iotalens.py

```diff
@@ -5,49 +5,45 @@
 # License: BSD-3-Clause-LBNL
 #
 
-import glob
-
 import numpy as np
-import pandas as pd
+import openpmd_api as io
 from scipy.stats import moment
 
 
 def get_moments(beam):
-    """Calculate mean and std dev of functions defining the IOTA invariants
-
-    Returns
-    -------
-    meanH, sigH, meanI, sigI
-    """
-    meanH = np.mean(beam["H"])
-    sigH = moment(beam["H"], moment=2) ** 0.5
-    meanI = np.mean(beam["I"])
-    sigI = moment(beam["I"], moment=2) ** 0.5
-
-    return (meanH, sigH, meanI, sigI)
-
-
-def read_all_files(file_pattern):
-    """Read in all CSV files from each MPI rank (and potentially OpenMP
-    thread). Concatenate into one Pandas dataframe.
```

Comment:
We should discuss what to do about this output & analysis capability.  It is important, but only when using the IOTA nonlinear lens element.  A similar issue occurs for the multipole element.

Reply:
Agreed. I think we can make it a custom reduced diagnostics.

We should add a custom input control for it, so it is only run when we request it.